rr_viz/README.md

```diff
@@ -3,6 +3,9 @@
     This package is an 'advanced' GUI for RR purposes. The GUI is made from various  widgets put together (check individual widget read mes for details).
     The intention is to create widgets as needed and compose the GUI in a an easy way using QT designer/editor
 
+# Install/Dependencies
+- Must have rr_custom_msgs built and available
```

Comment:
Since this package is open source is it time to open source rr_custom_msgs too?

Reply:
Yeah i think so, i pressume we need to give it a license in the package.xml though